### PR TITLE
[Merged by Bors] - chore: another removal of unused variables

### PR DESCRIPTION
**Pull request by adomani**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -20,5 +20,19 @@
...
+    "latex",
+    "plaintext"
+  ],
+  "spellright.parserByClass": {
+    "lean4": {
+      "parser": "code",
+      "comment_start": "/-",
+      "comment_end": "-/",
+      "comment_line": "--"
+    }
```
Comment (Ruben-VandeVelde) [inline .vscode/settings.json:36]: Did you mean to add this?

File: Mathlib/Algebra/Group/Pi/Lemmas.lean [hunk 2]
```diff
@@ -27,7 +27,7 @@ variable {I : Type u}
 variable {f : I → Type v}
 
 -- The family of types already equipped with instances
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Group/Pi/Lemmas.lean:29]:
```suggestion
```
no?

File: Mathlib/Algebra/Group/Submonoid/Defs.lean [hunk 3]
```diff
@@ -47,15 +47,12 @@ submonoid, submonoids
 assert_not_exists CompleteLattice
 assert_not_exists MonoidWithZero
 
--- Only needed for notation
 -- Only needed for notation
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Group/Submonoid/Defs.lean:50]: I don't understand what this is about, so maybe
```suggestion
```

File: Mathlib/Algebra/Group/Subsemigroup/Defs.lean [hunk 4]
```diff
@@ -48,12 +48,11 @@ assert_not_exists CompleteLattice
 assert_not_exists MonoidWithZero
 
 -- Only needed for notation
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Group/Subsemigroup/Defs.lean:50]:
```suggestion
```

File: Mathlib/Algebra/Ring/Pi.lean [hunk 5]
```diff
@@ -26,7 +26,7 @@ variable {I : Type u}
 variable {f : I → Type v}
 
 -- The family of types already equipped with instances
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Ring/Pi.lean:28]:
```suggestion
```

**Review by Ruben-VandeVelde — APPROVED** [review]
_So much_ cargo culting

File: .vscode/settings.json (hunk 1 above)
Comment (adomani) [reply .vscode/settings.json:36]: Oops!  Thanks for noticing!

**Comment by adomani** [comment]
> _So much_ cargo culting

Indeed, so much of it, that I never know if I simply forgot to PR some changes and I am repeating code pruning over and over again!

I think that this arises a lot when people split files.

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**